src/components/Header/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Header.component';
```

Comment:
This pattern is okay, but a little complicated.
You usually have `index.jsx`when you are going to export a lot of components/sub-components from a folder.
Because it makes sense to have an `index` ... like books
If not, you can just have a `Header.jsx` inside the `Header` folder and import it as `import Header from /Header/Header.jsx`. You can also have the component inside the index.jsx file to just do `import Header from /Header`. You'll come across all these conventions. I've never seen `./Header.component`, though.